test_conformance/c11_atomics/common.h

```diff
@@ -281,6 +283,92 @@ class CBasicTest : CTest
     else
       return 0;
   }
+
+  int CheckCapabilities(TExplicitMemoryScopeType memoryScope,
+                        TExplicitMemoryOrderType memoryOrder)
+  {
+      /*
+          Differentiation between atomic fence and other atomic operations
+          does not need to occur here.
+
+          The initialisation of this test checks that the minimum required
+          capabilities are supported by this device.
+
+          The following switches allow the test to skip if optional capabilites
+          are not supported by the device.
+        */
+      switch (memoryScope)
+      {
+          case MEMORY_SCOPE_EMPTY: {
+              break;
+          }
+          case MEMORY_SCOPE_WORK_GROUP: {
+              if ((gAtomicMemCap & CL_DEVICE_ATOMIC_SCOPE_WORK_GROUP) == 0)
+              {
+                  return TEST_SKIPPED_ITSELF;
+              }
+              break;
+          }
+          case MEMORY_SCOPE_DEVICE: {
+              if ((gAtomicMemCap & CL_DEVICE_ATOMIC_SCOPE_DEVICE) == 0)
+              {
+                  return TEST_SKIPPED_ITSELF;
+              }
+              break;
+          }
+          case MEMORY_SCOPE_ALL_SVM_DEVICES: {
+              if ((gAtomicMemCap & CL_DEVICE_ATOMIC_SCOPE_ALL_DEVICES) == 0)
+              {
```

Comment:
Super minor: there appears to be an extra set of `{` braces `}` here.  Also below, for `MEMORY_ORDER_SEQ_CST`.